src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/ActivitiesReconService.kt

```diff
@@ -69,19 +69,27 @@ class ActivitiesReconService(
     )
   }
 
-  private fun publishTelemetry(
+  private suspend fun publishTelemetry(
     prisonId: String,
     compareResults: CompareBookingsResult,
   ) {
     val telemetryName =
```

Comment:
not sure why this is needed as a variable since there is an `if` block below anyway